middleware/browse/browse.go

```diff
@@ -241,6 +241,11 @@ func (b Browse) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error) {
 		if !middleware.Path(r.URL.Path).Matches(bc.PathScope) {
 			continue
 		}
+		switch r.Method {
+		case http.MethodGet, http.MethodHead:
+		default:
+			return http.StatusMethodNotAllowed, nil
+		}
```

Comment:
Oh, sorry, didn't catch that in time.

It's because we will likely want to add methods OPTIONS and PROPFIND here; optionally with some additional checks.

And, generally speaking, it's better to use ```switch``` on sets that ```if```.